tests/apitests/robot/cmd.c

```diff
@@ -523,14 +523,29 @@ static void killnode(TestContext *context, int argc, char *argv[])
 static void restartnode(TestContext *context, int argc, char *argv[])
 {
     CarrierContextExtra *extra = context->carrier->extra;
+    CarrierContext *wctx = context->carrier;
     struct timeval timeout_interval;
     struct timeval now;
     extern void *carrier_run_entry(void *);
 
-    CHK_ARGS(argc == 2 || argc == 3);
+    CHK_ARGS(argc == 1 || argc == 2 || argc == 3);
 
     vlogI("Robot will be reborn.");
 
+    if (argc == 1) {
```

Comment:
Here (argc == 1) would cover the later situation: 
if (argc == 2) {
    extra->test_offmsg = OffMsgCase_Single;
} else {
    // cover the situation when argc == 1
     .....
}

Reply:
If execution steps into this block, it returns at the end, never steps out of the block.